src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+abstract class Machine {

```

Comment:
The `Machine` class should be public to ensure that it can be accessed from other packages if needed.